config/routes.rb

```diff
@@ -1,16 +1,28 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :topics
-  resources :organizers do
+
+  scope '/admin' do
+    resources :topics, only: [:edit, :new]
+    resources :organizers, only: [:edit, :new]
+    resources :locations, only: [:edit, :new]
+    resources :opportunities, only: [:edit, :new]
+    resources :opportunity_instances, only: [:edit, :new]
+  end
+
+  resources :topics, except: [:edit, :new]
+  resources :locations, except: [:edit, :new]
+  resources :opportunities, except: [:edit, :new]
+
+  resources :organizers, except: [:edit, :new] do
     resources :opportunities
```

Comment:
I don't think the admin side will need it so i'm going to go with it if it looks good when i boot it up